SourcetoSeperationstoStoragetoPlant_v1.xml

```diff
@@ -0,0 +1,176 @@
+  </archetypes>
+
+    <commodity>
+    <name>Tritium</name>
+    <solution_priority>1.0</solution_priority>
+  </commodity>
+  <commodity>
+    <name>Enriched_Lithium</name>
+    <solution_priority>1.0</solution_priority>
+  </commodity>
+  <commodity>
+    <name>Helium_3</name>
+    <solution_priority>1.0</solution_priority>
+  </commodity>
+
+   <recipe>
+    <name>Lithium</name>
+    <basis>atom</basis>
+    <nuclide>
+      <id>30060000</id>
+      <comp>0.3</comp>
+    </nuclide>
+    <nuclide>
+      <id>30070000</id>
+      <comp>0.7</comp>
+    </nuclide>
+  </recipe>
+
+    <recipe>
+    <name>Tritium</name>
+    <basis>atom</basis>
+    <nuclide>
+        <id>10030000</id>
+        <comp>1</comp>
+    </nuclide>
+  </recipe>
+
+<facility>
+    <name>PlantOne</name>
+      <config>
+      <FusionPowerPlant>
+        <fusion_power>100</fusion_power>
+        <TBR>1.1</TBR>
+	     <reserve_inventory>6.0</reserve_inventory>
+     	<!--<startup_inventory>8.121</startup_inventory> -->
+        <sequestered_equilibrium>2.121</sequestered_equilibrium>
+        <fuel_incommod>Tritium</fuel_incommod>
+        <blanket_incommod>Enriched_Lithium</blanket_incommod>
+        <blanket_inrecipe>Lithium</blanket_inrecipe>
+        <blanket_size>1000</blanket_size>
+        <he3_outcommod>Helium_3</he3_outcommod>
+	    <Li7_contribution>1.0</Li7_contribution>
+        <refuel_mode>fill</refuel_mode>
+        <buy_frequency>5</buy_frequency>
+	     <blanket_outcommod>Tritium</blanket_outcommod>
+        <blanket_turnover_fraction>0.1</blanket_turnover_fraction>
+        <blanket_turnover_frequency>6</blanket_turnover_frequency>
+      </FusionPowerPlant>
+    </config>
+  </facility>
+
+  <facility>
+    <name>One Source</name>
+    <config>
+        <Source>
+            <outcommod>Tritium</outcommod>
+            <outrecipe>Tritium</outrecipe>
+            <throughput>2.133</throughput>
+        </Source>
+    </config>
+  </facility>
+
+  <facility>
+    <name>One Storage</name>
+    <config>
+      <Storage>
+        <in_commods>
+            <val>Tritium</val>
+        </in_commods>
+        <out_commods>
+            <val>Tritium</val>
+        </out_commods>
+        <throughput>0.210833</throughput>
+      </Storage>
+    </config>
+  </facility>
+
+ <facility>
+  <name>Separations Facility</name>
+  <config>
+    <Separations>
+      <feed_commods>
+        <val>Tritium</val>
+      </feed_commods>
+      <feed_commod_prefs>
+        <val>1.0</val>
+        <val>1.0</val>
+      </feed_commod_prefs>
+      <feedbuf_size>8.121</feedbuf_size>
+      <throughput>1e+3</throughput>
+      <leftover_commod>Helium_3</leftover_commod>
+      <leftoverbuf_size>1e+3</leftoverbuf_size>
+      <streams>
+        <item>
+          <commod>Separated_Material</commod>
+          <info>
+            <buf_size>5e+3</buf_size>
+            <efficiencies>
+              <item>
+                <comp>2003000</comp>
+                <eff>0.99</eff>
```

Comment:
How does Tricycle deal with potentially getting feed that has a little He3 in it?